neuro_san/__init__.py

```diff
@@ -0,0 +1,19 @@
+
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+from leaf_common.config.file_of_class import FileOfClass
+
+# Normally we don't use __init__.py files to define anything,
+# but here we define some constants that point to important directories in the distribution.
+TOP_LEVEL_DIR = FileOfClass(__file__)
+DEPLOY_DIR = FileOfClass(__file__, path_to_basis="./deploy")
+REGISTRIES_DIR = FileOfClass(__file__, path_to_basis="./registries")
```

Comment:
Add some constants to the top-level `__init__.py` file so we do not have to have so many FileOfClass instances floating around the code base.  This gets to be a problem when those files within the codebase move around.
Better to have a few constants at the top and have stuff be relative references to those.

Besides the TOP_LEVEL_DIR, the other 2 constants are here merely because they are used more than once throughout the code base, so it's reasonable to put them up here.